batches/cloudFormation/batches.yaml

```
@@ -0,0 +1,62 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Resources:
+  BatchLayer:
+    Type: AWS::Lambda::LayerVersion
+    Properties:
+      CompatibleRuntimes:
+        - nodejs14.x
+      Content:
+        S3Bucket: caliber-batches
+        S3Key: BatchesLayersCode
+      Description: BatchesLayersCode
+      LayerName: batches-layers
+  Batches:
+    Type: AWS::Lambda::Function
+    Properties:
+      FunctionName: Batches
+      Handler: lambda/index.handler
+      Layers:
+        - !Ref BatchLayer
+      Role:
+        Fn::GetAtt:
+          - BatchesRole
+          - Arn
+      Code:
+        S3Bucket: caliber-batches
+        S3Key: BatchesCode
+      Runtime: nodejs14.x
+      Timeout: 300
```

maybe a 5 second timeout instead of a 5 minute timeout?
```suggestion
      Timeout: 5
```